mods/tuxemon/l18n/fr_FR/LC_MESSAGES/base.po

```diff
@@ -1118,12 +1124,12 @@ msgstr "Pas vraiment."
 
 # NPC dialogs
 msgid "professor_dialog\n"
-msgstr "Professor:\n"
-"Ah, yes. The very person I've been wanting to meet.\n"
-"I've heard that you want to learn about Tuxemon.\n"
-"Tuxemon are little animals, some call them monsters, that we befriend and use to battle other Tuxemon.\n"
-"I was wondering, I just received three new Tuxemon for research and I would like to give you one.\n"
-"Which one would you like?\n"
+msgstr "Professeur:\n"
+"Tu es celui que j'attendais.\n"
+"On m'a dit que tu voulais découvrir les Tuxemon.\n"
+"Les Tuxemon sont des petits animaux (ou \"monstres\"), que l'on amadoue et que l'on fait se battre contre d'autres Tuxemon.\n"
```

Comment:
Wondering if there would not be something better than "amadoue"
"domestique" ? "capture" ? "dresse" ?

I don't know if these suggestions would really sound better, I'm just giving alternatives.

Reply:
Since the English version said humans befriend Tuxemon it made
more sense to me to use « amadouer »